backend/user_management/minio/routers/upload.py

```
@@ -0,0 +1,41 @@
+from fastapi import APIRouter, UploadFile, File, HTTPException, Depends
+from sqlalchemy.orm import Session
+from typing import List
+from user_management.database import get_db
+from user_management.models import Product
+from user_management.minio.utils import upload_to_minio
+from user_management.minio.schemas import ProductPhotoUploadResponse, ProductVideoUploadResponse
+
+router = APIRouter()
+
+@router.post("/upload/photos/{product_id}", response_model=ProductPhotoUploadResponse)
+async def upload_photos(
```

Ruff (0.8.2)

Missing return type annotation for public function `upload_photos`

(ANN201)